impeller/display_list/canvas.cc

```diff
@@ -1084,6 +1119,42 @@ void Canvas::SaveLayer(const Paint& paint,
         transform_stack_.back().transform.HasTranslation()
             ? Entity::RenderingMode::kSubpassPrependSnapshotTransform
             : Entity::RenderingMode::kSubpassAppendSnapshotTransform);
+
+    if (will_cache_backdrop_texture) {
+      FML_DCHECK(backdrop_data);
+      // If all filters on the shared backdrop layer are equal, process the
+      // layer once.
+      if (backdrop_data->all_filters_equal &&
+          !backdrop_data->filtered_input_slot.has_value()) {
+        // TODO(jonahwilliams): compute minimum input hint.
+        backdrop_data->filtered_input_slot =
```

Comment:
Ahh, so `filtered_input_slot` is the shared output across all the equal filters.  I don't quite know what a "slot" means in this case.  Usually i think of slot meaning an attachment point for a shader.

Should this maybe be called something like `shared_equal_filter_output`?

Reply:
How about shared_filter_snapshot?

Reply:
sounds good to me